util/stop/stopper_test.go

```diff
@@ -15,13 +15,19 @@
 //
 // Author: Spencer Kimball (spencer.kimball@gmail.com)
 
-package util
+package stop
 
 import (
 	"testing"
 	"time"
+
+	"github.com/cockroachdb/cockroach/testutils/fakeflags"
 )
 
+func init() {
+	fakeflags.InitLogFlags()
```

Comment:
Rather than an `init` function which calls `fakeflags.InitLogFlags`, you could place an `init` function in the `fakeflags` package and then force link the package above using `import _ "github.com/cockroachdb/cockroach/testutils/fakeflags"`